File: CodeForces/Java/CF_66B_PetyaAndCountryside.java
//  Problem: 66B. Petya and Countryside.
//  Link: https://codeforces.com/problemset/problem/66/B

import java.util.*;

class Main {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);
    
    int n = scan.nextInt();
    
    int[] numbers = new int[n];
    for (int i = 0; i < n; i++) {
        numbers[i] = scan.nextInt();
    }
    
    int maxWatered = 0;

    for (int i = 0; i < n; i++){
      int currentWater = 1;
        
      for (int j = i; j < n - 1; j++){
        if (numbers[j] >= numbers[j+1]){
          currentWater++;
        } else{
          break;
        }
      }
      
      for (int j = i; j > 0; j--){
        if (numbers[j] >= numbers[j-1]){
          currentWater++;
        } else{
          break;
        }
      }
      
      if (currentWater > maxWatered) {
          maxWatered = currentWater;
      }
    }
    
    System.out.println(maxWatered);
    
    scan.close();
  }
}

File: CodeForces/Python/CF_66B_PetyaAndCountryside.py
# Problem: 66B. Petya and Countryside.
# Link: https://codeforces.com/problemset/problem/66/B

n = int(input())

numbers = list(map(int, input().split()))

max_watered = 0

for i in range(len(numbers)):
  current_water=1
  for j in range(i,len(numbers)-1):
    if(numbers[j]>=numbers[j+1]):
      current_water+=1
    else:
      break
  for j in range(i,0,-1):
    if(numbers[j]>=numbers[j-1]):
      current_water+=1
    else:
      break
  if(current_water>max_watered):
    max_watered=current_water
print(max_watered)